VideoWeb/VideoWeb/ClientApp/src/app/shared/models/hearing.ts

```diff
@@ -10,8 +10,10 @@ import { HearingBase } from './hearing-base';
 import { Participant } from './participant';
 
 export class Hearing extends HearingBase {
+    _participants: Participant[];
```

Comment:
I think this array should probably be private as it looks like it's only used within the scope of the component and not in the view. Also not sure how I feel about _name convention can it be given a name that diffrentiates it from the other participants array?